config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -89,6 +89,8 @@ default double defaultTopKProbability() {
 
         String docprocLoadBalancerType();
 
+        String jvmGCOptions();
```

Comment:
Consider adding a default implementation (you'll have to do that later anyway) + adding a comment on its purpose.

Reply:
I think we will keep this flag for a very long time for rolling out chnages to GC  as we discover better combinations.